EIPS/eip-4788.md

```diff
@@ -80,7 +80,9 @@ When verifying a block, execution clients **MUST** ensure the root value in the
 #### Block processing
 
 At the start of processing any execution block where `block.timestamp >= FORK_TIMESTAMP` (i.e. before processing any transactions),
-write the parent beacon root provided in the block header into the storage of the contract at `HISTORY_STORAGE_ADDRESS`.
+
+- write the parent beacon root provided in the block header into the storage of the contract at `HISTORY_STORAGE_ADDRESS`.
+- `iff` the `HISTORY_STORAGE_ADDRESS` has `nonce==0`, set `nonce` to `1`. This is to prevent touch-delete rules from deleting the account from the state.
```

Comment:
I am not understanding the usage and formatting of this `iff`.

iff -> if ?
And is there a reason to have this in parenthesis?

Reply:
Respectively:

iff -> If would rather be the change I guess with a capital `I` since starting a new sentence.

Reply:
iff denotes "if and only if". I used `code formatting` to denote that it was an exact technical term, but maybe it's just superfluous, and a classic "if" works just as well. 

```suggestion
- If the `HISTORY_STORAGE_ADDRESS` has `nonce==0`, set `nonce` to `1`. This is to prevent touch-delete rules from deleting the account from the state.
```